src/util/PathUtil.ts

```diff
@@ -165,3 +165,23 @@ export function createSubdomainRegexp(baseUrl: string): RegExp {
   const { scheme, rest } = extractScheme(baseUrl);
   return new RegExp(`^${scheme}(?:([^/]+)\\.)?${rest}`, 'u');
 }
+
+/**
+ * Returns the folder corresponding to the root of the Community Solid Server module
+ */
+export function getModuleRoot(): string {
+  return joinFilePath(__dirname, '../../');
```

Comment:
Would there be cases where we also want to make this be configurable from the CLI? (I guess not?)

Reply:
I can't really think of a use case. Users can always configure with full paths if they need something special. This is mostly just so we can safely reference other files within the package.